src/app/lorax/directives/modal-email.tpl.html

```diff
@@ -6,16 +6,15 @@
         <h1>{{::modalEmail.content.title}}</h1>
       </header>
       <p class="keep-informed__copy keep-informed__body-text">{{::modalEmail.content.copy}}</p>
-      <form id="enter-email" class="keep-informed__form" method="post" action="https://sendto.mozilla.org/page/s/sign-up-for-mozilla" ng-submit="modalEmail.onFormSubmit()">
+      <form id="enter-email" class="keep-informed__form" method="post" ng-submit="modalEmail.onFormSubmit()">
         <div class="enter-email__user-input cf">
           <div class="enter-email__email">
             <label for="email" class="is-vishidden">{{::modalEmail.content.label}}</label>
             <input name="email" type="email" id="email" autocorrect="off" autocapitalize="off" spellcheck="false" required aria-required="true" placeholder="{{::modalEmail.content.label}}" ng-model="modalEmail.email" ng-change="modalEmail.onInputChange()">
           </div>
           <div class="enter-email__country">
-            <select>
-              <option>United States</option>
-              <option ng-repeat="country in ::modalEmail.countries">{{country.displayName}}</option>
+            <select name="country" ng-model="modalEmail.country">
```

Comment:
@schalkneethling any idea how I can select an item by default? I think the proper answer is to set the default in the controller here. It should be USA.